docs/pull_requests.md

```diff
@@ -0,0 +1,89 @@
+# Pull Request process
+
+Pull Requests in the `openshift-ansible` project follow a
+[Continuous](https://en.wikipedia.org/wiki/Continuous_integration)
+[Integration](https://martinfowler.com/articles/continuousIntegration.html)
+process that is similar to the process observed in other repositories such as
+[`origin`](https://github.com/openshift/origin).
+
+Whenever a
+[Pull Request is opened](../CONTRIBUTING.md#submitting-contributions), some
+automated test jobs must be successfully run before the PR can be merged.
+
+Some of these jobs are automatically triggered, e.g., Travis and Coveralls.
+Other jobs need to be manually triggered by a member of the
+[@team-openshift-ansible-contributors](https://github.com/orgs/openshift/teams/team-openshift-ansible-contributors) team.
+
+## Triggering tests
+
+We have two different Jenkins infrastructures, and, while that holds true, there
+are two commands that trigger a different set of test jobs. The objective is to
+move towards a single infrastructure, simplifying the workflow.
```

Comment:
"The objective is to...." seems out of place, or like it's missing something. The beginning of the paragraph describes two different sets of tests. And then this says the objective is to have a single one. Maybe more like this?

```
We have two different Jenkins infrastructures, and, while that holds true, there
are two commands that trigger a different set of test jobs. We are working on 
simplifying the workflow towards a single infrastructure in the future.
```